components/CampaignListItem/index.tsx

```diff
@@ -0,0 +1,121 @@
+import ArchiveOutlinedIcon from '@mui/icons-material/ArchiveOutlined'
+import { Box } from '@mui/material'
+
+const styledmainbox = {
+  display: 'flex',
+  flexDirection: 'column',
+  width: '1070px',
+  height: '100%',
+  padding: '10px',
+  backgroundColor: '#EAF9FF',
+}
+
+const styledmainbody = {
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'space-between',
+  padding: '0px 20px 0px 20px',
+}
+
+const styledcontentbox = {
+  display: 'flex',
+  height: '300px',
+  backgroundColor: '#D9D9D9',
+}
+
+const styledbottombox = {
+  display: 'flex',
+  flex: 'auto',
+  flexDirection: 'row',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  width: '100%',
+  height: '67px',
+  backgroundColor: '#FFFFFF',
+  padding: '0px 20px 0px 20px',
+}
+
+const styledbottomitems = {
+  display: 'flex',
+}
+
+const styledinput = {
+  fontSize: '15',
+  fontWeight: '700',
+}
+
+type Props = {
+  CampaignName: String
+  CampaignDetails: String
+  Donors: Number
+  Amount: Number
+  Country: String
+  Published: Boolean
+  ViewAll: Boolean
+}
+
+const CampaignListItem = ({
+  CampaignName,
+  CampaignDetails,
+  Donors,
+  Amount,
+  Country,
+  Published,
+  ViewAll,
+}: Props) => {
+  return (
+    <Box sx={styledmainbox}>
+      <Box sx={styledmainbody} className="main-body">
+        <div
+          style={{
+            display: 'flex',
+            justifyContent: 'space-between',
+            alignItems: 'center',
+            flexDirection: 'row',
+          }}
+        >
+          <h3 style={{ color: '#009DD7' }}>{CampaignName}</h3>
+          <ArchiveOutlinedIcon
+            sx={{
+              color: '#009DD7',
+              fontSize: '2.5rem',
+            }}
+          />
+        </div>
+        <div
+          style={{
+            display: 'flex',
+            justifyContent: 'space-between',
+            flexDirection: 'row',
+          }}
+        >
+          <p>{CampaignDetails}</p>
+          <h4
+            style={{
+              color: Published ? '#16A64A' : '#C7043D',
+            }}
+          >
+            {Published ? 'Published' : 'Draft'}
+          </h4>
+        </div>
+      </Box>
+      <div>{ViewAll ? <Box sx={styledcontentbox}></Box> : <p></p>}</div>
```

Comment:
this box should contain campaign details
`{ViewAll && <Box sx={styledcontentbox}>{CampaignDetails}</Box>}`

currently it looks like this. can help to fix the padding for this grey box
<img width="983" alt="Screen Shot 2022-12-11 at 1 40 23 AM" src="https://user-images.githubusercontent.com/51109678/206868135-48ad70f6-6fba-4fa0-8197-51de4f5853f8.png">

Reply:
![image](https://user-images.githubusercontent.com/7589432/206893218-58c6d9d5-b779-42fd-96e2-0664a7bdf689.png)

Made changes, current render